site/sfguides/src/kepler_gl_maps_inside_snowflake/kepler_gl_maps_inside_snowflake.md

```diff
@@ -0,0 +1,290 @@
+author: Vladi Bilonenko
+id: kepler_gl_maps_inside_snowflake
+summary: This guide will help you create interactive maps directly within Snowflake using Kepler.gl
+categories: Getting-Started
+environments: web
+status: Published
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, Maps, Kepler.gl
+
+# Creating Interactive Maps with Kepler.gl in Snowflake Using Dekart
+<!-- ------------------------ -->
+## Overview
+Duration: 1
+
+This guide will help you create interactive maps directly within Snowflake using [Kepler.gl](https://kepler.gl), powered by [Dekart Snowpark Application](https://dekart.xyz/docs/snowflake-snowpark/about/). With this approach, you can analyze and visualize geospatial data from public datasets, all within Snowflake, avoiding external data transfers and approvals.
+
+### What You Will Build
+
+A map that visualizes UK highways with color-coded density of nearby EV charging stations.
+
+![UK highways with color-coded density of nearby EV charging stations](assets/kepler_gl_snowflake_1.png)
+
+### What You Will Learn
+- How to create interactive maps within Snowflake using Kepler.gl and Dekart.
+- How to access and use Overture Maps data to build your map.
+- How to construct SQL queries for extracting, filtering, and visualizing geospatial data.
+
+### Prerequisites
+
+**Snowflake Account** with `ACCOUNTADMIN` role for managing permissions.
```

Comment:
@sfc-gh-obielov Link Snowflake account to signin/signup page.

Reply:
Added